src/docdb/nodes.ts

```diff
@@ -152,4 +164,37 @@ export class DocDBDocumentNode implements INode {
 		arguments: [this],
 		title: ''
 	};
+}
+
+export class LoadMoreNode implements INode {
+	constructor(readonly iterator: QueryIterator<any>, readonly parentNode: DocDBCollectionNode) {
+	}
+
+	readonly id = `${this.parentNode.id}.LoadMore`;
+
+	readonly label = `Load More...`;
+
+	readonly contextValue = 'LoadMoreButton'
+
+	readonly command: Command = {
+		command: 'cosmosDB.loadMore',
+		arguments: [this],
+		title: ''
+	};
+
+	static async loadMore(iterator: QueryIterator<any>, batchSize: number = 20): Promise<IResults> {
```

Comment:
I still feel this is a weird place to put this logic. I think the 'Load More' node should be a very simple wrapper for the 'cosmosDB.loadMore' command just like the SignInToAzureNode. And all state/functions should be moved to the DocDBCollectionNode node.

That being said, we're going to refactor anyways when we add LoadMore in other places - so you don't have to change for this PR if you don't want to.